baremetal-coco/install.sh

```diff
@@ -197,6 +264,32 @@ function add_image_pull_secret() {
 #    $label
 function create_runtimeclasses() {
     local tee_type=${1}
+
+    case $tee_type in
+        tdx)
+            echo "kata-tdx | creating runtime class"
+
+            oc apply -f - <<EOF
+apiVersion: node.k8s.io/v1
+kind: RuntimeClass
+metadata:
+  name: kata-tdx
+handler: kata-tdx
+overhead:
+  podFixed:
+    memory: "350Mi"
+    cpu: "250m"
+    tdx.intel.com/keys:  1
+scheduling:
+  nodeSelector:
+     intel.feature.node.kubernetes.io/tdx:  true
```

Comment:
I think you'll need to have `node-role.kubernetes.io/kata-oc: ""'` also in the selector. This label indicates that kata is installed.